publisher/image/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.1
 
+RUN apk add --update-cache curl && rm -rf /var/cache/apk/*
```

Comment:
Ahh, I get the motivation behind this now. Would you mind adding a comment to this tune so we know why we're adding this (for future reference)? :)